src/round.rs

```diff
@@ -181,9 +181,6 @@ where
             return Err(RoundingError::DurationExceedsLimit);
         }
         let stamp = naive.timestamp_nanos_opt().ok_or(RoundingError::TimestampExceedsLimit)?;
-        if span > stamp.abs() {
-            return Err(RoundingError::DurationExceedsTimestamp);
-        }
```

Comment:
If you remove this check, I expect it will introduce a panic if the date is large and the duration is even larger. Say a date in the year 200.000 and rounding to a duration of 300.000 years.

Could you add a test for this case, and do something to avoid the panic?

Reply:
I couldn't find any. Added a test for timestamps that are close to the min / max of nanosecond precision

Reply:
Thank you 👍. @djc will also want to have a look, but it seems good to merge for me.